languages/go/exercises/concept/numbers-floating-point/numbers-floating-point_test.go

```diff
@@ -0,0 +1,200 @@
+package numbers_floating_point
+
+import (
+	"testing"
+)
+
+func TestAnnualBalanceUpdate(t *testing.T) {
+	tests := []struct {
+		name    string
+		balance float64
+		want    float64
+	}{
+		{
+			name:    "AnnualBalanceUpdateForEmptyStartBalance",
```

Comment:
We can use spaces in these names, which would make them a bit easier to read, wouldn't it?